src/queries/voting-data.ts

```diff
@@ -20,45 +20,50 @@ const aragonEnv = {
  * @param id - id of the vote to query (I.E. 21)
  * @param votingContractAddress - voting contract to query -- defaults to the main sarcophagus DAO address
  */
-function formatVoteId(id: string | number, votingContractAddress: string = '0xf483c1f7858dd19915d0689d26cb3487fc90b640'): any {
-  const aragonVoteIdPrefix = `appAddress:${votingContractAddress}-vote:`
-  const aragonVoteId = aragonVoteIdPrefix + "0x" + Number(Number(id).toString(16)).toString(16)
-  return aragonVoteId
+function formatVoteId(
+  id: string | number,
+  votingContractAddress: string = '0xf483c1f7858dd19915d0689d26cb3487fc90b640'
+): any {
+  const aragonVoteIdPrefix = `appAddress:${votingContractAddress}-vote:`;
+  const aragonVoteId = aragonVoteIdPrefix + '0x' + Number(Number(id).toString(16)).toString(16);
+  return aragonVoteId;
 }
 
+/**
+ * Retrieves details of vote including address of voters and timing of execution
+ *
+ * @param web3 - blockchain provider
+ * @param voteId - id of the vote to query (I.E. 21)
+ */
 export async function fetchVoteData(web3: any, voteId: string | number): Promise<VotingData> {
-  const voteIdFormatted = formatVoteId(voteId)
+  const voteIdFormatted = formatVoteId(voteId);
 
   // Connect to aragon subgraph to retrieve voting data
-  const org = await connect(
-    aragonEnv.location,
-    'thegraph', 
-    { 
-      network: aragonEnv.network,
-      ethereum: web3.currentProvider
-    }
-  )
+  const org = await connect(aragonEnv.location, 'thegraph', {
+    network: aragonEnv.network,
+    ethereum: web3.currentProvider,
+  });
 
   // Grab the first 100 votes
   // TODO: will need to consider pagination as vote count exceeds 100
-  const voting = await connectVoting(org.app('voting'))
-  const votes = await voting.votes({ first: 100 })
+  const voting = await connectVoting(org.app('voting'));
```

Comment:
For some reason the `connectVoting()` function returns type `any`. This can be cast to the `Voting` type provided by aragon: 
```
const voting: Voting = await connectVoting(org.app('voting'));
```

You may import this type using an auto import tool (`ctrl+.` in vscode), or add this `import { Voting } from '@aragon/connect-voting';`